Library/Formula/b2-tools.rb

```diff
@@ -0,0 +1,15 @@
+class B2Tools < Formula
+  desc "B2 Cloud Storage Command-Line Tools"
+  homepage "https://www.backblaze.com/b2/docs/quick_command_line.html"
+  url "https://docs.backblaze.com/public/b2_src_code_bundles/b2"
+  version "0.3.12"
+  sha256 "b50576006739c592333d13cbde9a21cbb3b7e2ba3b99bfedbd70a7adbe097aa8"
+
+  def install
+    bin.install "b2"
```

Comment:
Hi,
This is a self contained script. There's not much that I can do for testing because all other features require authentication. Do you have any ideas on a suitable test? 





On Fri, Feb 5, 2016 at 11:57 AM -0800, "Mike McQuaid" <notifications@github.com> wrote:












In Library/Formula/b2-tools.rb:
> @@ -0,0 +1,15 @@
> +class B2Tools < Formula
> +  desc "B2 Cloud Storage Command-Line Tools"
> +  homepage "https://www.backblaze.com/b2/docs/quick_command_line.html"
> +  url "https://docs.backblaze.com/public/b2_src_code_bundles/b2"
> +  version "0.3.12"
> +  sha256 "b50576006739c592333d13cbde9a21cbb3b7e2ba3b99bfedbd70a7adbe097aa8"
> +
> +  def install
> +    bin.install "b2"



Is this a binary or script?



—
Reply to this email directly or view it on GitHub.